dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/HostFactoryImpl.java

```diff
@@ -116,7 +117,8 @@ public class HostFactoryImpl implements HostFactory {
             .SYSTEM_HOST).append("' ");
 
     private static final StringBuilder SITE_IS_LIVE = new StringBuilder().append("cvi.live_inode IS NOT NULL");
-    private static final StringBuilder SITE_IS_LIVE_OR_STOPPED = new StringBuilder().append("cvi.live_inode IS NOT null or " +
+    @VisibleForTesting
+    protected static final StringBuilder SITE_IS_LIVE_OR_STOPPED = new StringBuilder().append("cvi.live_inode IS NOT null or " +
```

Comment:
does not make sense to have constant stringbuilder
since it is mutable object anyhow and not thread-safe

Reply:
Suggested changes were implemented https://github.com/dotCMS/core/pull/25442/commits/2fbd6f1a3ea607741ce371421151039ddc4d67cb